community/import-tool/src/test/java/org/neo4j/tooling/EntityDataGenerator.java

```diff
@@ -84,7 +84,7 @@ public long position()
     public void close()
     {
         super.close();
-        processing.shutdown( false );
+        processing.shutdown( 0 );
```

Comment:
Would be really nice if we will have plain `shutdown()` in case everything is fine and `shutdown(TaskPolicy taskPolicy)` that take enum value as param instead of int.
Then in simple case you do not need even think what the number you need to pass that and in case you actually need to pass something - you can see enum values instead of trying yo find int value